### DISCUSSION TITLE
Script to create large data nodes

### DISCUSSION THREAD

**Discussion by ShahanaFarooqui** [ACINQ/eclair#2396] [Ideas]
Is it possible to provide a script which can generate large transactions data for a node? 

I do not have much data on my node to test my application's performance for a larger node. The ability to generate a node with a lot of peers, channels, transactions can be very helpful in debugging performance issues & errors.

Rusty has also recently wrote a python script to do the same for Core Lightning [here](https://github.com/ElementsProject/lightning/pull/5549).

**Comment by t-bast** [comment]
That could be useful indeed. More generally, we could import some of the test scripts I have at https://github.com/t-bast/lightning-cfg/ in the `contrib` folder, that would help people experiment with eclair on regtest.
